app/models/grade.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Grade < ApplicationRecord
+  has_many :classroom_grades, dependent: :destroy
+  has_many :classrooms, through: :classroom_grades
+
+  validates :name, presence: true
```

Comment:
We have DB validation, which is excellent! 

But it does not give a helpful error message to the users.

Adding validation to the `name` and `level` is useful here. it can be in the follow-up ticket.